app/controllers/carto/api/paged_searcher.rb

```
@@ -39,6 +33,40 @@ def paged_result(result:, total_count:, page:, per_page:, order:)
         metadata[:_links][:next] = { href: yield(page: page + 1, per_page: per_page, order: order) } if last_page > page
         metadata
       end
+
+      def order_from_params(valid_order_values, default_order:, valid_order_combinations:)
+        order = (params[:order].presence || default_order).to_sym
+        order_array = order.to_s.split(',').map(&:to_sym)
```

Could we have repeated values for `order` in the request?

---

Yes, we could, but it works as expected, although it has no sense to do it with `order`. Do you think we should throw an error or something?

---

Do you know if this is going to be used as part of a public API or are we the only clients? If we are the only clients I guess it's fine going forward without any further validations.

---

It seems like it's not public